bower.json

```diff
@@ -16,9 +16,9 @@
     "marked": "~0.5",
     "MathJax": "^2.7.4",
     "moment": "~2.19.3",
-    "preact": "https://unpkg.com/preact@^7.2.0/dist/preact.min.js",
-    "preact-compat": "https://unpkg.com/preact-compat@^3.14.3/dist/preact-compat.min.js",
-    "proptypes": "https://unpkg.com/proptypes@^0.14.4/index.js",
+    "preact": "https://unpkg.com/preact@~7.2.0/dist/preact.min.js",
+    "preact-compat": "https://unpkg.com/preact-compat@~3.14.3/dist/preact-compat.min.js",
+    "proptypes": "https://unpkg.com/proptypes@~0.14.4/index.js",
```

Comment:
I'm hazy on this, what's the difference between `^` and `~`?

Reply:
I see this change was already merged in #4681, so never mind.

Reply:
Essentially, `~` is more restrictive. In this situation, `^` means include the latest minor release (7.x, 3.x, 0.14.x, respectively), while `~` means the latest patch release (7.2.x, 3.14.x, and 0.14.4, respectively). See examples at https://semver.npmjs.com/ (enter the package name and semver range, and it will show you matching versions)

Reply:
Thanks Jason!